projects/ngx-oxford/src/lib/oxford.pipe.ts

```diff
@@ -1,5 +1,10 @@
 import { Pipe, PipeTransform } from '@angular/core';
 
+interface TruncateOptions {
+  trail: string;
+  wordCount: number;
```

Comment:
Rename to maxCount or maxItemCount to be more clear.

Reply:
Done